### DISCUSSION TITLE
Battery profile on switch

### DISCUSSION THREAD

**Discussion by texelmicro** [iNavFlight/inav#8938] [Q&A]
Prob an odd question, is there any way to change battery profile with a switch on the transmitter? I have a very unique build that would be difficult to use stick commands or OSD menu. Any ideas will help. Thanks!

**Comment by Jetrell** [comment]
The programming framework provides a `Set_Profile` operation, you can program via a switch. https://github.com/iNavFlight/inav/blob/master/docs/Programming%20Framework.md
But it doesn't support `ControlRate_profile` for battery profile selection. As far as I'm aware, its primarily designed to be changed by the detected battery voltage.
